lib/converter/rule.go

```diff
@@ -187,9 +187,9 @@ func ruleActionAPIToBackend(action string) string {
 // value to the equivalent API value.
 func ruleActionBackendToAPI(action string) string {
 	if action == "" {
-		return "allow"
+		return "Allow"
```

Comment:
Not sure any changes are required in this file.  The converter code was used for the v1 API -> v1 backend conversion, but I don't think anything uses that anymore.  In any case, the action in the v1 API is lowercase, so this should need updating for these case changes.

Reply:
Yes, good catch: reverting this change and also a couple of changes I made to a v1 api test.